README.md

```diff
@@ -13,7 +13,8 @@ python: true
 require: 
   - ./readme.python.md
   - ./readme.cli.md
-  
+try-require:
+  - $(this-folder)/readme.az.common.md

```

Comment:
we shouldn't have readme.az.common.md

just readme.az.md and readme.cli.md

Reply:
Yes, will use readme.az.md

Reply:
Hi @zikalino, I notice readme.az.md/readme.cli.md need to be configured at each RP. 

This configuration feature should be a common config apply for all PR, shall we config it in this README.md? Otherwise,  readme.az.common.md seems should be a good location to do this global configuration.

Reply:
yes, README.md in autorest.az plugin is the right place if it  applies to all RPs

Reply:
readme.az.common.md is used for put all the RPs common settings. it's in autorest.az folder not in swagger